data/35e/wizards_of_the_coast/supplement/complete_scoundrel/cs_equip.lst

```diff
@@ -30,14 +30,14 @@ Trail Bar			TYPE:Alchemical	COST:20	WT:0		SOURCEPAGE:p.111	SPROP:This tough, 4-i
+Salvo			TYPE:Poison	COST:100	WT:0		SOURCEPAGE:p.113	SPROP:The fortuitous creation of a poisoner with little skill, salvo is a weapon of the imprecise and desperate. The poison is relatively cheap and easy to create, but its effect fluctuates wildly from dose to dose. Salvo deals 1d4 points of initial and secondary damage to a random ability score. Determine the affected ability by rolling 1d6 and consulting the following table; roll once per dose. The poison's type and save DC do not vary.
+Siren's Breath		TYPE:Poison	COST:300	WT:0		SOURCEPAGE:p.113	SPROP:This bitter-smelling gas causes the subject to become highly susceptible to persuasion. Failing either the initial or secondary saving throw imposes a -5 penalty on Sense Motive checks and on Will saves against enchantment spells and effects for 10 rounds. If a creature is subjected to another dose of siren's breath before the effect of the first dose has run its course, the duration of the first dose's effect ends.
+Slow Death		TYPE:Poison	COST:250	WT:0		SOURCEPAGE:p.113	SPROP:An insidiously persistent toxin distilled in part from a dwarven mining acid, slow death eats away at its victims from within. Unlike most poisons, which deal damage in two stages, slow death works constantly over the course of several rounds. A subject of this poison who fails the initial Fortitude save takes 1d6 points of damage. Each round thereafter, at the beginning of the poisoner's turn (up to a maximum of 5 rounds), the subject must repeat the saving throw. For each failure, it takes another 1d6 points of damage. Once a save succeeds, the subject takes no further damage.
+Zealot's Blade		TYPE:Poison	COST:350	WT:0		SOURCEPAGE:p.113	SPROP:This venom incorporates the blood of its intended target, making it even deadlier to such creatures. A tool of hunters and assassins who stalk specific targets, each dose of zealot's blade is brewed for a specific type or subtype of creatures. When creating zealot's blade, the poisonmaker must choose a type (and subtype, if applicable) from Table 3-14: Ranger Favored Enemies on page 47 of the Player's Handbook. Any creature not of the chosen type or subtype gains a +5 bonus on saves to resist the poison's effect.
```

Comment:
Mate,

Only change I see is Poisons > Poison. Can you explain what this is fixing?